vagrant/devmode/salt/roots/virtualenv.sls

```diff
@@ -29,6 +29,7 @@ pip_pkgs:
     - activate: true
     - requirements: /home/vagrant/calamari/requirements/2.7/requirements.txt
     - download_cache: /vagrant/pip_cache
+    - env_vars: SWIG_FEATURES=-cpperraswarn
```

Comment:
one thing that seems wrong about this approach is we would probably like to know if other SWIG dependent packages have errors that we transmute to warnings. i.e. this probably shouldn't apply as a blanket to all elements in requirements.txt